de.metas.adempiere.adempiere/base/src/main/java/de/metas/adempiere/service/impl/AddressBuilder.java

```diff
@@ -86,11 +88,10 @@ public AddressBuilder()
 	 * @param userBlock
 	 * @return
 	 */
-	public String buildAddressString(final I_C_Location location, boolean isLocalAddress, String bPartnerBlock, String userBlock)
+	public String buildAddressString(final I_AD_Org org, final I_C_Location location, final String language, boolean isLocalAddress, String bPartnerBlock, String userBlock)
```

Comment:
pls document if "org" is a mandatory parameter or it's an optional one.
I think it's an optional one so why adding it as first parameter?